Italian/main/Settings.apk/res/values-it/strings.xml

```diff
@@ -3244,26 +3244,26 @@ If you choose not to use personalized ads, the amount of advertising won't be re
     <string name="search_music_mi_effect_title">Mi Sound Enhancer</string>
     <string name="search_zen_mode_option_miui_silent">Silent mode</string>
     <string name="search_vibrate_when_ringing_title">Also vibrate for calls</string>
-    <string name="search_vibrate_in_silent_title">Vibrate when silent</string>
-    <string name="search_vibrate_on_touch_title">Vibrate on touch</string>
+    <string name="search_vibrate_in_silent_title">Vibra in silenzioso</string>
+    <string name="search_vibrate_on_touch_title">Vibra al tocco</string>
     <string name="search_dtmf_tone_enable_title">Dial pad touch tones</string>
-    <string name="search_sound_effects_enable_title">Touch sounds</string>
-    <string name="search_lock_sounds_enable_title">Screen lock sound</string>
+    <string name="search_sound_effects_enable_title">Suoni tocco</string>
+    <string name="search_lock_sounds_enable_title">Suoni Blocco schermo</string>
     <string name="search_unlock_set">Sblocco</string>
     <string name="search_screen_timeout">Disattivazione schermo</string>
-    <string name="search_owner_info_settings">Owner info</string>
+    <string name="search_owner_info_settings">Info utente</string>
     <string name="search_security_passwords_title">Password</string>
-    <string name="search_turn_on_security_lock">Change screen lock</string>
-    <string name="search_wakeup_for_keyguard_notification_title">Wake lock screen for notifications</string>
-    <string name="search_pref_volume_wake_title">Wake with Volume buttons</string>
-    <string name="search_screen_on_proximity_sensor_title">Prevent pocket dials</string>
+    <string name="search_turn_on_security_lock">Cambia Blocco schermo</string>
```

Comment:
Prima di sparare a zero, assicurati di quello che stai dicendo. 
Ci sono 18 ricorrenze per "Blocco schermo" in questo file.

<img width="215" alt="schermata 2016-08-05 alle 20 16 48" src="https://cloud.githubusercontent.com/assets/10470552/17446432/95234c50-5b49-11e6-9124-51ef47ed93bd.png">